_sources/Iteration/NestedIterationImageProcessing.rst

```diff
@@ -18,13 +18,13 @@ Two dimensional tables have both rows and columns. You have probably seen many t
 spreadsheet program. Another object that is organized in rows and columns is a digital image. In this section we will
 explore how iteration allows us to manipulate these images.
 
-A **digital image** is a finite collection of small, discrete picture elements called **pixels**. These pixels are 
```

Comment:
These changes should not be included in this PR as they are only changing whitespace at the end of each line.  That just creates confusion and unnecessary conflicts.  Please remove these files from the PR.